Ginger/GingerCoreNET/Run/RunListenerLib/CenteralizedExecutionLogger/AccountReportExecutionLogger.cs

```diff
@@ -71,14 +71,23 @@ public async Task SendRunsetExecutionDataToCentralDbTaskAsync(RunSetConfig runse
 
         public async Task RunSetEnd(RunSetConfig runsetConfig)
         {
+
             await SendRunsetEndDataToCentralDbTaskAsync(runsetConfig);
+
             Reporter.HideStatusMessage();
         }
 
         public async Task SendRunsetEndDataToCentralDbTaskAsync(RunSetConfig runsetConfig)
         {
             AccountReportRunSet accountReportRunSet = AccountReportEntitiesDataMapping.MapRunsetEndData(runsetConfig, mContext);
-            await AccountReportApiHandler.SendRunsetExecutionDataToCentralDBAsync(accountReportRunSet, true);
+            bool isDataUploadedOnCentralDb = await AccountReportApiHandler.SendRunsetExecutionDataToCentralDBAsync(accountReportRunSet, true);
+
+            LiteDBRepository liteDB = new();
```

Comment:
Also this object is present under Execution logger manager, please try referring it from there

Reply:
Will consider this to redesign the code.